test/kubectx.bats

```diff
@@ -87,3 +104,81 @@ load common
   echo "$(get_context)"
   [[ "$(get_context)" = "user2@cluster1" ]]
 }
+
+@test "get one context and switch to non existent context" {
+  use_config config1
+
+  run ${COMMAND} "unknown-context"
+  echo "$output"
+  [ "$status" -eq 1 ]
+}
+
+@test "rename context" {
+  use_config config2
+
+  run ${COMMAND} "new-context=user1@cluster1"
+  echo "$output"
+  [ "$status" -eq 0 ]
+
+  run ${COMMAND}
+  echo "$output"
+  [ "$status" -eq 0 ]
+  [[ "$output" = *"new-context"* ]]
+  [[ "$output" = *"user2@cluster1"* ]]
```

Comment:
isn't the regexp syntax supposed to be `=~`?
why does this even work 🤔

Reply:
hmm interesting I guess this is just bash glob feature without regexps. in that case it's fine.

```
[[ "ahmet" = *"ahm"* ]]; echo $?
0
```